backend-stack-bubbling/app.py

```diff
@@ -547,6 +554,46 @@ def post():
                 "is_upvote": info["is_upvote"]
             }), 200)
 
+class DeclareBestAnswer(Resource):
+    @staticmethod
+    @jwt_required()
+    def post():
+        info = BestAnswerInfo.parse_args()
+        identity = get_jwt_identity()
+        responseMessage = ""
+        currentUser = UserCollection.find_one(
+            {
+                "email": identity["email"]
+            })
+        questionID = uuid.UUID(info["question_id"])
+        answerID = uuid.UUID(info["answer_id"])
+        if currentUser is not None:
+            answer = QuestionCollection.find_one(
+                {
+                    "_id": questionID,
+                    
+                }
+            )
+            print('here be answer')
+            print(answer)
+            QuestionCollection.update(
```

Comment:
Can a question have two best answers? Because it does not check if there is already a best answer